openfecli/commands/gather.py

```diff
@@ -229,10 +229,10 @@ def gather(rootdir, output, report):
 
     \b
     * 'dg' (default) reports the ligand and the results are the maximum
-      likelihood estimate of its absolute free, and the uncertainty in
-      that.
+      likelihood estimate of its absolute free, and the associated
+      uncertainty from DDG replica averages and standard deviations.
```

Comment:
"derived from the network of" or something like that? Totally not past Balmer's peak